chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json

```diff
@@ -0,0 +1,54 @@
+{
+	"info": {
+		"_postman_id": "21c26d32-3ea4-4d99-ac3d-c4c557baa398",
+		"name": "developer-controller",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "GET /developers",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "89af904b-9c4d-4c17-9fd1-83ff5d945beb",
+						"exec": [
+							"pm.test(\"Invalid api key header should return 401 status\", function () {\r",
+							"    pm.response.to.have.status(401);\r",
```

Comment:
Is there any reason some controllers have both valid and invalid api test and some dont. Example, this developer controller has only invalid api key test. Why not add valid api key test?
Status controller has both valid and invalid api key tests, which is good

Reply:
fixed